mods/ra/rules/defaults.yaml

```diff
@@ -491,6 +491,8 @@
 	Inherits@1: ^ExistsInWorld
 	Inherits@2: ^IronCurtainable
 	Inherits@3: ^SpriteActor
+	BodyOrientation:
+		CameraPitch: 255
```

Comment:
This changes the shape of armament muzzles as they rotate with facing.  This was previously tuned to look correct, so I am assuming that this will break that.  Unless this was specifically to fix that?